### PR TITLE
Feat/sales recap --run-build

### PR DESCRIPTION
**Pull request by jokomyn**
changelog:
- sales recap integrate
- merge table
- integrate search

### CODE REVIEW

**Comment by akarintitech** [comment]
# SonarQube Code Analytics 
## Quality Gate Status: OK

passed

[Sonar report](https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=18) (https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=18)

## Additional information
*The following metrics might not affect the Quality Gate status but improving them will improve your project code quality.*

## Issues
- Bugs: security_a [0 Bugs](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=BUG&pullRequest=18)

- Vulnerabilities: security_a [0 Vulnerabilities](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=VULNERABILITY&pullRequest=18)

- Code Smells: security_a [0 Code Smells](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=CODE_SMELL&pullRequest=18)

- Security Hotspots: security_a [0 Security Hotspots](https://sonar.akarinti.tech/security_hotspots?id=dayacore_pos_web&pullRequest=18&inNewCodePeriod=true)



## Coverage and Duplications
- Coverage: **Coverage**

- Duplications:  [0.00% Duplication](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&metric=new_duplicated_lines_density&view=list&pullRequest=18)

File: src/app/components/Features/MenuList/MenuList.component.tsx [hunk 1]
```diff
@@ -328,6 +334,34 @@ const MenuList: React.FC<menuProps> = ({data='', detailBook='', onChangeAddInfo,
...
+    tmpMenu.map((item:IItem)=>{
+      if(item.id === selectedMenu.id) {
+        
+        // delete item['item_extra']
+        // delete item['item_package']
+      }
+    })
+    setSelectedMenuMenu(null)
+    // setTmpItemExtra(undefined)
+    // setTmpItemPackage([])
```
Comment (aldiz1711) [inline src/app/components/Features/MenuList/MenuList.component.tsx:361]: Bisa di delete jika sudah tidak dipakai

File: src/app/components/Features/Package/Package.component.tsx [hunk 2]
```diff
@@ -139,19 +185,21 @@ const PackageComponent: React.FC<PackageProp> = ({menuItem}) => {
                                 size='small'
                                 data-testid='next-button'
                                 onClick={()=>{
-                                    item.qty = item.qty ? item.qty + 1 : 1;
+                                    // console.log(selectedGroup);
+                                    handleSelectMenu(item, 'ADD')
+                                    // setSelectedGroup(selectedGroup)
```
Comment (aldiz1711) [inline src/app/components/Features/Package/Package.component.tsx:190]: Bisa di delete jika sudah tidak dipakai

**Comment by akarintitech** [comment]
# SonarQube Code Analytics 
## Quality Gate Status: OK

passed

[Sonar report](https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=18) (https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=18)

## Additional information
*The following metrics might not affect the Quality Gate status but improving them will improve your project code quality.*

## Issues
- Bugs: security_a [0 Bugs](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=BUG&pullRequest=18)

- Vulnerabilities: security_a [0 Vulnerabilities](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=VULNERABILITY&pullRequest=18)

- Code Smells: security_a [0 Code Smells](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=CODE_SMELL&pullRequest=18)

- Security Hotspots: security_a [0 Security Hotspots](https://sonar.akarinti.tech/security_hotspots?id=dayacore_pos_web&pullRequest=18&inNewCodePeriod=true)



## Coverage and Duplications
- Coverage: **Coverage**

- Duplications:  [0.00% Duplication](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&metric=new_duplicated_lines_density&view=list&pullRequest=18)

**Pull request merged**